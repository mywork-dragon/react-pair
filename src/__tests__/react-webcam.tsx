import * as React from 'react';
import * as renderer from 'react-test-renderer';
import Webcam from '../react-webcam';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <Webcam
        audio={false}
        audioConstraints={{
          sampleSize: 8,
          echoCancellation: true
        }}
        className="react-webcam"
        imageSmoothing={false}
        minScreenshotHeight={1000}
        minScreenshotWidth={1000}
        onUserMedia={() => {}}
        onUserMediaError={() => {}}
        screenshotFormat="image/png"
        screenshotQuality={1}
        style={{transform: 'rotate(180deg)'}}
        videoConstraints={{
          width: 160,
          height: 120,
          frameRate: 15
        }}
      />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
