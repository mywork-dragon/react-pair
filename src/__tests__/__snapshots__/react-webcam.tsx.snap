// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<video
  autoPlay={true}
  className="react-webcam"
  muted={false}
  playsInline={true}
  style={
    Object {
      "transform": "rotate(180deg)",
    }
  }
/>
`;
